ansible/configs/satellite-multi-region/software.yml

```diff
@@ -11,13 +11,9 @@
   hosts: satellites
   become: True
   gather_facts: True
-  pre_tasks:
-      - name: Copy /etc/hosts file
-        template:
-          src: "./files/etc_hosts_template.j2"
-          dest: /etc/hosts
   roles:
-    - { role: "satellite-installation",                 when: install_satellite }   
+    - { role: "satellite-installation",                 when: install_satellite   } 
+    - { role: "satellite-manage-organization",          when: configure_satellite }  
     - { role: "satellite-manage-manifest",              when: configure_satellite } 
     - { role: "satellite-manage-subscription-and-sync", when: configure_satellite } 
```

Comment:
Suggest changing this to the 'satellite-manage-subscription' role. The initial sync takes an eternity, and is not desired in most cases.

Reply:
considering your suggestion, I have renamed role and limit the functionality to subscription only . Also deleted satellite-v64-prod.  Thanks